app/Resources/assets/js/admin.js

```diff
@@ -0,0 +1,69 @@
+var $ = require('jquery');
+window.$ = $;
+window.jQuery = $;
+
+require('bootstrap-sass');
+require('moment');
```

Comment:
This shouldn't be needed. I think only the `eonasdan-bootstrap-datetimepicker` library uses it, and it should (and appears to) require `moment` internally on its own https://github.com/Eonasdan/bootstrap-datetimepicker/blob/25c11d79e614bc6463a87c3dd9cbf8280422e006/src/js/bootstrap-datetimepicker.js#L42